Surface_mesh_simplification/include/CGAL/Surface_mesh_simplification/Policies/Edge_collapse/Edge_profile.h

```diff
@@ -76,9 +76,9 @@ public :
   Edge_profile(const halfedge_descriptor h_v0v1,
                TM& tmesh,
                const Geom_traits& traits,
-               const VertexIndexMap& vim,
-               const VertexPointMap& vpm,
-               const HalfedgeIndexMap& him,
+               const VertexIndexMap vim,
```

Comment:
why changing that?

Reply:
Just harmonization of the way property maps are handled